docs/rest_api.rst

```diff
@@ -663,3 +663,51 @@ be rejected.
   :statuscode 404: If the channel does not exist
   :statuscode 409: If the given block number argument is invalid
   :statuscode 500: Internal Raiden node error
+  
+
+.. http:get:: /api/1/channels/history/(token_address)/(partner_address)
+
+   Query for all internal events tied to a specific token network channel. If the partner is not provided, the client will provide a list containing all events for all channels.
```

Comment:
> internal events tied to a specific token network channel

This endpoint is not for internal events but for channel history. It returns only 3 events which are user facing.